lib/importers/google/satisfaction/call_record.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+module Importers
+  module Google
+    module Satisfaction
+      class CallRecord
+        SATISFACTION_VALUE = {
+          'Delighted' => 4,
+          'Vey pleased' => 3,
```

Comment:
Good call, this came from the documentation and I never thought to check it was required. 

Removed.